presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -446,6 +447,9 @@ private static boolean isPrestoQueryInvalid(SQLException e)
                 if (object instanceof Array) {
                     object = ((Array) object).getArray();
                 }
+                if (object instanceof byte[]) {
+                    object = new SqlVarbinary((byte[]) resultSet.getObject(i));
```

Comment:
instead of calling `resultSet.getObject(i)` again, just use the `object` variable like `new SqlVarbinary((byte[]) object)`